tests/localLock.test.js

```diff
@@ -85,7 +85,7 @@ describe('local locks', function () {
                 resolve();
               }
             }, 100);
-          })();
+          })().then(() => {});
```

Comment:
I'm not that familiar with this particular code, so I'd appreciate a second set of eyes here.